modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java

```diff
@@ -270,6 +270,12 @@ private InfoFieldSet _getBasicInformationInfoFieldSet() {
 			ObjectEntryInfoItemFields.userNameInfoField
 		).infoFieldSetEntry(
 			ObjectEntryInfoItemFields.userProfileImageInfoField
+		).infoFieldSetEntry(
```

Comment:
Hey @ericthmoritsuka, could you please sort this?

Reply:
and done. thx @seltong